client/src/store/new-request-module.ts

```diff
@@ -2180,9 +2183,9 @@ export class NewRequestModule extends VuexModule {
     try {
       const { nrId } = this
       const nr = this.editNameReservation
-      const requestData = await this.addRequestActionComment(nr)
+      const requestData = nr && await this.addRequestActionComment(nr)
 
-      const response = await axios.patch(`/namerequests/${nrId}/edit`, requestData, {
+      const response = requestData && await axios.patch(`/namerequests/${nrId}/edit`, requestData, {
```

Comment:
What happens to the code below (eg, line 2194) if response is undefined (because requestData is falsy)?